### ISSUE TITLE
Improper tag match in JSX

### ISSUE THREAD

**Issue by mike3run** [morhetz/gruvbox#179]
![screen shot 2017-07-30 at 12 11 07 pm](https://user-images.githubusercontent.com/7088511/28755422-37f30b82-7520-11e7-95d6-f56675f37a20.png)

This happened as well on the Oceanic-Next package and here is the solution that was made to fix it maybe it will help in this case as well?

https://github.com/mhartington/oceanic-next/commit/267726a293a4f305d44fe39708df0d367b8b6e97

**Comment by mike3run** [comment]
So after some digging i see a more straightforward fix for the dark scheme in normal contrast would be 

```vim
hi xmlTagName guifg=#fbf1c7
```

Which ends up rendering like this:

![screen shot 2017-08-05 at 2 46 57 pm](https://user-images.githubusercontent.com/7088511/28998295-f157efb8-79ec-11e7-848a-6bd23311aaa3.png)

I checked the sourcecode but it seems like you are doing some really clever things to render color  so a more elegant solution is needed for it to be merged, but at least for my use case i can just pop that in my vimrc and be done with it.

Im still leaving this open in case anyone wants to figure this out a bit better?

**Comment by morhetz** [comment]
@mike3run it seems like it's the problem with oceanic custom xml syntax
I'm 99% sure it would display tags all blue if you comment oceanic plug at your init.vim
Tried to reproduce locally with vanilla gruvbox and different js-syntaxes but it's cool

**Comment by mike3run** [comment]
@morhetz i dont have oceanic next installed as of right now, and the bug still exists its okay since i already did a custom patch for my case but just letting you know about the coloring issue :)

**Comment by mike3run** [comment]
the only xml syntax thing i use is this one

https://github.com/sheerun/vim-polyglot

**Comment by morhetz** [comment]
@mike3run 
mind sharing your vimrc and vim version?

**Comment by mike3run** [comment]
@morhetz 

I have a fix you can implement for this in 1 line 😄 

`hi! link xmlTagN GruvboxBlue`

**Comment by mike3run** [comment]
@morhetz made a PR with the fix #187

**Comment by mike3run** [comment]
@morhetz with the fix I added on the PR the tags would look like this:

<img width="300" alt="screen shot 2017-09-21 at 1 49 11 pm" src="https://user-images.githubusercontent.com/7088511/30713072-be4a6bb2-9ed3-11e7-8552-2c50ed7104b9.png">


The minimal vimrc would be:

```vim
Plug 'morhetz/gruvbox'
Plug 'sheerun/vim-polyglot'

let g:gruvbox_italic = 1
colorscheme gruvbox
set background=dark
```
